Esp32PmSensor/Esp32PmSensor.ino

```diff
@@ -19,13 +19,12 @@
 
 #include "sds011.h"
 
-// This EUI must be in BIG-ENDIAN format, so least-significant-byte first.
+// This EUI must be in BIG-ENDIAN format, most-significant byte (MSB).
+// In TTN console switch format with the <> button and choose msb with ⇄
 // For TTN issued EUIs the first bytes should be 0x70, 0xB3, 0xD5.
 static const u1_t APPEUI[8] = { 0x70, 0xB3, 0xD5, 0x7E, 0xD0, 0x01, 0xA0, 0x9B };
 
-// This key should be in big endian format (or, since it is not really a
-// number but a block of memory, endianness does not really apply). In
-// practice, a key taken from ttnctl can be copied as-is.
+// This key should be in big endian format as well, see above.
```

Comment:
Thanks, much better!

Reply:
Thanks for your work. Let me know if I can help testing with Heltec/BME280/SDS011.